cmd/remove.go

```diff
@@ -10,17 +10,24 @@ package cmd
 
 import (
 	"github.com/spf13/cobra"
+	"github.com/vanilla-os/orchid/cmdr"
 )
 
-func NewRemoveCommand() *cobra.Command {
-	cmd := &cobra.Command{
-		Example: "apx remove htop",
-		Use:     "remove <packages>",
-		Short:   "Remove packages inside a managed container.",
-		RunE:    remove,
-	}
-	cmd.Flags().BoolP("assume-yes", "y", false, "Proceed without manual confirmation.")
-
+func NewRemoveCommand() *cmdr.Command {
+	cmd := cmdr.NewCommand("remove <packages>",
+		apx.Trans("remove.long"),
+		apx.Trans("remove.short"),
+		remove).WithBoolFlag(
+		cmdr.NewBoolFlag(
```

Comment:
[This issue](https://github.com/Vanilla-OS/orchid/issues/13) in Orchid may cause this flag to conflict with other `--assume-yes`